README.md

```diff
@@ -117,6 +117,7 @@ Then run the following commands to run the DB migrations:
 # Install project tools
 dotnet tool restore
 # Run migrations
+# You'll need to cd into the project you want to run the datbase migration for - either LeaderboardBackend or LeaderboardBackend.Test
```

Comment:
```suggestion
cd LeaderboardBackend
```
Migrations are only run in that project anyway, either we say this or we add the project flag to the database update command below.